sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -326,8 +319,8 @@ def estimate_size(self, unused_value, nested=False):
 
 class TimestampCoderImpl(StreamCoderImpl):
 
-  def __init__(self, timestamp_class):
-    self.timestamp_class = timestamp_class
+  def __init__(self, timestamp_class=None):
+    self.timestamp_class = Timestamp#timestamp_class
```

Comment:
Left over comment `#timestamp_class` ?

Actually is there a need for the `timestamp_class` argument any more?